boards/arduino-leonardo/leonardo-runner.sh

```diff
@@ -28,4 +57,4 @@ read -r
 printf "Attempting to flash ...\n"
 printf "\n"
 
-avrdude -qq -C/etc/avrdude.conf -patmega32u4 -cavr109 -P/dev/ttyACM0 -b57600 -D "-Uflash:w:$1:e"
```

Comment:
I'll have to try whether removing `-C/etc/avrdude.conf` works for me as well (I expect it will but better to double check ...).  Apart from that, this looks good now!

One more thing: Can you please do the same changes for the runner scripts for the other supported boards as well?  I know, this is a bit ugly to have code duplicated all over the place, but as I already mentioned the current runner scripts aren't here to stay anyway ...

Reply:
Done 👍